### PR TITLE
UI

### PR DESCRIPTION
**Pull request by OscarFMdev**
# PR description

In this PR I:
- Created a terminal app 💻 
- Added colors to the app. 🌈 
- Fixed linter errors 🤖 

### Thank you for reviewing my code! 🙌🏽

### CODE REVIEW

File: app.rb [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+    message = @books.empty? && "\nNo books added, please, add some books \n".colorize(:red)
+    puts message
+    @books.each do |book|
+      puts "Title: #{book.title}, Author: #{book.author}".colorize(:green)
+    end
+  end
+
+  def list_people
+    message = @people.empty? && "\nNo people added, please, add some people \n".colorize(:red)
+    puts message
```
Comment (danmainah) [inline app.rb]: - [ ] @OscarFMdev great work on your project however this message  `puts message`  gets displayed even when the list is not empty making the output look like there is an error kindly make it render only when the list is empty,
image

File: app.rb [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      end
+      # if_condition
+    end
+    # loop
+  end
+  # selection
+
+  def list_books
+    message = @books.empty? && "\nNo books added, please, add some books \n".colorize(:red)
+    puts message
```
Comment (danmainah) [inline app.rb]: - [ ] @OscarFMdev great work on your project however this message `puts message` gets displayed even when the list is not empty making the output look like there is an error kindly make it render only when the list is empty,
image

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @OscarFMdev , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
✔️ No linter errors
✔️ Gitflow was correctly used
✔️  Good design


# Required Changes ♻️

- [ ] kindly remove the getting started section at the top of your readme file since you already have another getting started section

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by OscarFMdev** [comment]
@danmainah Thank you for taking a good look at my code! 🙌🏽

File: app.rb (hunk 1 above)
Comment (Whoistolu) [reply app.rb]: Well resolved @OscarFMdev 👍🏽

**Review by Whoistolu — APPROVED** [review]
### Hi @OscarFMdev 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

# STATUS: APPROVED 🟢
**Congratulations! 🎉** 

- [ ] [OPTIONAL] Well done @OscarFMdev , I see that you added the `colorize` dependency to beautify the UI. Kindly add this to the README that the dependency should be installed locally by running `bundle install` or running
```
gem install colorize
```

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by OscarFMdev** [comment]
@Whoistolu Thank you! I'll add it to my readme file

**Pull request merged**